auslib/db.py

```diff
@@ -1764,6 +1765,56 @@ def _putWatchdogValue(self, changed_by, value, where=None, transaction=None):
             super(Dockerflow, self).update(where=where, what=value, changed_by=changed_by, transaction=transaction)
 
 
+class Shutoffs(AUSTable):
+    def __init__(self, db, metadata, dialect):
+        self.table = Table('shutoffs', metadata,
+                           Column('shutoff_id', Integer, primary_key=True,
+                                  autoincrement=True),
+                           Column('product', String(15), nullable=False),
+                           Column('channel', String(75)),
+                           Column('mapping', String(100)),
+                           UniqueConstraint('product', 'channel')
```

Comment:
I'm also wondering if there's a use case for mapping to be NULL. If we can't think of one, it's probably best to set nullable=False for it.